libraries/chain/webassembly/eos-vm-oc/LLVMJIT.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "Inline/BasicTypes.h"
+#include "IR/Module.h"
+
+#pragma push_macro("N")
+#undef N
```

Comment:
Nothing in this file uses ``N``, you can just #undef it.

Reply:
llvm's headers do
Just commenting out the `#undef N` gives me errors like

```
In file included from /home/spoon/eos/src/libraries/chain/webassembly/eos-vm-oc/LLVMJIT.h:8:
In file included from /usr/include/llvm/IR/Module.h:27:
In file included from /usr/include/llvm/IR/Function.h:28:
In file included from /usr/include/llvm/IR/BasicBlock.h:23:
In file included from /usr/include/llvm/IR/Instruction.h:22:
In file included from /usr/include/llvm/IR/DebugLoc.h:18:
In file included from /usr/include/llvm/IR/TrackingMDRef.h:17:
/usr/include/llvm/IR/Metadata.h:1227:48: error: member initializer 'string_to_name' does not name a non-static data member or base class
  MDTupleTypedArrayWrapper(const MDTuple *N) : N(N) {}
                                               ^~~~
/home/spoon/eos/src/libraries/chain/include/eosio/chain/name.hpp:89:28: note: expanded from macro 'N'
#define N(X) eosio::chain::string_to_name(#X)
```

Reply:
The #undef is definitely needed.  It's the push and pop that are unnecessary.

Reply:
This is one of those cases where I prefer to leave things how I found them. Maybe some day it'll save someone a few minutes trying to figure out where `N()` is after LLVMJIT.h somehow gets included in their source file 3 layers deep